x-pack/metricbeat/module/aws/aws.go

```diff
@@ -208,6 +208,8 @@ func InitEvent(regionName string, accountName string, accountID string) mb.Event
 	if accountID != "" {
 		event.RootFields.Put("cloud.account.id", accountID)
 	}
+
+	event.Timestamp = timestamp
```

Comment:
I 've seen in the past that this kind of assignment was not working, could you confirm? 
Nevertheless I would go with sth like:
```
event := mb.Event{
  Timestamp: timestamp,
  RootFields: common.MapStr{},
```
wdyt?

Reply:
Good idea! Done!